fields-flow/src/commonMain/kotlin/dev/icerock/moko/fields/flow/BaseFlowFormField.kt

```diff
@@ -65,6 +65,15 @@ abstract class BaseFlowFormField<D, E>(
         showValidationError.value = true
         return isValid.value
     }
+
+    override fun clear() {
```

Comment:
i think we should remove `clear` fun - anybody can implement this by self just set data value and call `resetValidation`.
but currenct `clear` implementation have unexpected logic - not remove all data from input but set `initialValue`, that not expected for name `clear`